lib/LaTeXML/Engine/TeX_Box.pool.ltxml

```diff
@@ -368,8 +368,8 @@ sub insertBlock {
   my $inline    = $is_svg || $document->canContain($context_tag, '#PCDATA');
   my $container = $document->openElement('ltx:_CaptureBlock_', '_vertical_mode_' => 1, %blockattr);
   $document->absorb($contents);
-  my @nodes     = $container->childNodes;
-  my @node_tags = map { $document->getNodeQName($_); } @nodes;
+  my @nodes     = grep { $_->nodeType != XML_COMMENT_NODE; } $container->childNodes;
```

Comment:
should we make a new method akin to [`element_nodes`](https://github.com/brucemiller/LaTeXML/blob/d3d3fcc8d84cbcd789dbdd626effb1c786bbf771/lib/LaTeXML/Common/XML.pm#L92) that gets "material" child nodes?
Actually that may be easier with an accept-list rather than a reject-list, as in

```perl
sub content_nodes {
  my ($node) = @_;
  return ($node ? grep { my $t = $_->nodeType; $t == XML_ELEMENT_NODE || $t == XML_TEXT_NODE } 
    $node->childNodes : ()); }
```

Reply:
Good suggestion!